config/environments/test.rb

```diff
@@ -28,9 +28,6 @@
   config.consider_all_requests_local       = true
   config.action_controller.perform_caching = false
 
-  # Raise exceptions instead of rendering exception templates
-  config.action_dispatch.show_exceptions = false
-
```

Comment:
@pjanik, Won't this make it harder to see what went wrong when looking at a test console output?